### ISSUE TITLE
Examples of how to use TS from command line?

### ISSUE THREAD

**Issue by jpflorido** [iontorrent/TS#37]
Hi,
I have Ion Proton data (exome sequencing) and I have no idea how to analyze them. I've analyzed exome data from other platforms using the standard GATK pipeline, but I've not found documentation of how to analyze Ion Proton data (steps needed) from command line. 
The following document is suitable for users that have Torrent Suite from an user interface perspective:
https://assets.thermofisher.com/TFS-Assets/LSG/manuals/MAN0017203_TorrentSuite5_6_Help.pdf

Any help please for command line users?
Thanks,
Javier

**Comment by cweiming** [comment]
@jpflorido,

Have you find any document for command line users?

--
WM

**Comment by jpflorido** [comment]
No, no clues. This is why I do not like Ion Torrent (and do not recommend it).

**Comment by cweiming** [comment]
It's annoying! Almost all people recommended me to use TS for iontorrent samples due to its special core technology. However, no everyone has a torrent service to run torrent data.

Help ion torrent company can release an interactive interface of TS.

--
WM
